app/assets/javascripts/discourse/components/pwa-install-banner.js.es6

```diff
@@ -0,0 +1,53 @@
+import {
+  default as computed,
+  on
+} from "ember-addons/ember-computed-decorators";
+
+const userDismissedPromptKey = "dismissed-pwa-install-banner";
+
+export default Ember.Component.extend({
+  deferredInstallPromptEvent: null,
+
+  @on("init")
+  _registerListener() {
+    var that = this;
+    window.addEventListener("beforeinstallprompt", e => {
+      // Prevent Chrome 76+ from automatically showing the prompt
+      e.preventDefault();
+      // Stash the event so it can be triggered later
+      that.set("deferredInstallPromptEvent", e);
+    });
+  },
+
+  @computed
+  bannerDismissed: {
+    set(value) {
+      localStorage.setItem(userDismissedPromptKey, value);
+      return localStorage.getItem(userDismissedPromptKey);
+    },
+    get() {
+      return localStorage.getItem(userDismissedPromptKey);
+    }
+  },
+
+  @computed("deferredInstallPromptEvent", "bannerDismissed")
+  showPWAInstallBanner() {
+    return (
+      this.currentUser && // User must be logged in
+      this.currentUser.trust_level > 0 && // Be at least trust_level 1
```

Comment:
I think this comment is not necessary. `this.currentUser.trust_level > 0` seems very clear that we are checking we are above trust_level 1